schemas/spdx-schema.json

```diff
@@ -217,7 +218,9 @@
       "description" : "The URI provides an unambiguous mechanism for other SPDX documents to reference SPDX elements within this SPDX document."
     },
     "documentDescribes" : {
-      "description" : "Packages, files and/or Snippets described by this SPDX document",
+      "description" : "DEPRECATED: use relationships instead of this field. Packages, files and/or Snippets described by this SPDX document",
```

Comment:
How about "use CONTAINS relationships from the SPDXRef-DOCUMENT to the SPDX ID's in the documentDescribes list"?

Reply:
Nevermind - I see the `$comment` below which does have the specifics